src/panes.ts

```diff
@@ -142,6 +142,8 @@ export type FieldsPane = Pane<
       title: string
       provider?: ProviderMetadata
     }
+    help_link?: string
+    help_text?: string
```

Comment:
sorry, I meant have this on `AnyField` so we can attach help to multiple fields if necessary

Reply:
Ah ok, that can also get rid of the need for the `brivo_auth` flag

https://github.com/seamapi/seamapi-types/pull/73/commits/5c605e840e3f654c780ee622f306a56c881705e5

Reply:
oh nice